.lint-ignore

```diff
@@ -1,4 +1,6 @@
 accountEmails_4
+consumeRecoveryCode_2
```

Comment:
nit: `.lint-ignore` doesn't identify what it is for. is this `.mysql-lint-ignore`? there might be a better name for this

Reply:
Fair point, will update.